convert2rhel/actions/pre_ponr_changes/handle_packages.py

```diff
@@ -96,13 +160,39 @@ def run(self):
         logger.task("Convert: Remove packages containing .repo files")
         logger.info("Searching for packages containing .repo files or affecting variables in the .repo files:\n")
         try:
-            pkgs_to_remove = pkghandler._get_packages_to_remove(system_info.repofile_pkgs)
-            pkghandler.remove_pkgs_unless_from_redhat(pkgs_to_remove)
+            pkgs_to_remove = sorted(pkghandler._get_packages_to_remove(system_info.repofile_pkgs))
+            pkgs_removed = sorted(pkghandler.remove_pkgs_unless_from_redhat(pkgs_to_remove))
```

Comment:
This might need the same fix: 

```suggestion
            # this call can return None, which is not ideal to use with sorted.
            pkgs_removed = sorted(pkghandler.remove_pkgs_unless_from_redhat(pkgs_to_remove) or [])
```